website/scripts/build-static-content.js

```diff
@@ -720,6 +720,49 @@ module.exports = {
         }
         builtStaticContent.pricingTable = pricingTableCategories;
       },
+      async()=>{
+        builtStaticContent.ritualsTables = {};
+        // Find all the files in the top level /handbook folder and it's sub-folders
+        let FILES_IN_HANDBOOK_FOLDER = await sails.helpers.fs.ls.with({
+          dir: path.join(topLvlRepoPath, '/handbook'),
+          depth: 3
+        });
+        // Filter the list of filenames to get the rituals YAML files.
+        let ritualTablesYamlFiles = FILES_IN_HANDBOOK_FOLDER.filter((filePath)=>{
+          return _.endsWith(filePath, 'rituals.yml');
+        });
+        // Process each rituals YAML file. These will be added to the builtStaticContent as JSON
+        for(let ritualsYamlFilePath of ritualTablesYamlFiles){
+          // Get this rituals.yml file's parent folder name, we'll use this as the key for this section's rituals in the ritualsTables dictionary
+
+          let parentFolderOfThisYamlFile = ritualsYamlFilePath.split('handbook/')[1].split('/rituals.yml')[0];// TODO: get the parent folder name a better way.
```

Comment:
might be able to do `path.resolve(ritualsYamlFilePath, '../')`

Otherwise: 
`.split('/rituals.yml')` => `.split('rituals.yml')`  (you won't necessarily have the leading slash, since we don't look for it on 732)